scripts/notify-suppliers-of-brief-withdrawal.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+"""
+If a brief has been withdrawn suppliers need to be notified that their applications have also been cancelled. This
+script notifies suppliers of all briefs withdrawn on a given date. Alternatively, supplier with a given brief
```

Comment:
`Alternatively, supplier with a given brief...` this kind of trails off? 😸

Reply:
Fixed this